backend/src/db/pipelines/compliance/AllbbRequirements.ts

```diff
@@ -0,0 +1,34 @@
+export const aggregationPipeline = (): any[] => [
+    // Sort by bbName, bbKey, bbVersion, and dateOfSave descending
+    { $sort: { bbName: 1, bbKey: 1, bbVersion: 1, dateOfSave: -1 } },
```

Comment:
I think we should change the sorting, I believe that with current solution every new version would be after the first one, e.g. 
identity, bb-identity, 1.4, 2023-10-04
identity, bb-identity, 1.8, 2023-12-04
...
and latest one wouldn't be passed to the frontend when needed.
Can you confirm this and change code if necessary?

Reply:
Thanks, you are right. I will change the code to use ```Javascript { $sort: { bbName: 1, bbKey: 1, dateOfSave: -1 } },``` instead